form/models.py

```diff
@@ -6,6 +6,17 @@ class Form(models.Model):
 
     created = models.DateTimeField(auto_now_add=True)
     data = JSONField()
+    # Unique constraint to achieve “exactly-once delivery” with Amazon SQS
+    sqs_message_id = models.CharField(
+        max_length=255, blank=False, null=True, unique=True
```

Comment:
just to explain: you cannot have not-null as there is data in db already. Default cannot be blank, cause unique. I'd rather not assign random ids in data migration